.gitignore

```diff
@@ -4,3 +4,12 @@ build
 .idea
 *.iml
 
+//web
```

Comment:
curious: why two slashes?

Reply:
it's a comment, should I remove the comment or change it to `# comment`